src/app/(auth)/signup/page.tsx

```diff
@@ -1,36 +1,56 @@
 'use client';
 
 import { useState } from 'react';
-import { useRouter } from 'next/navigation';
+// import { useRouter } from 'next/navigation';
 import TextInput from '@/components/TextInput/index';
-import { H1, H4, AColored } from '@/styles/text';
+import { H1, H2, H4, AColored, P } from '@/styles/text';
 import supabase from '@/api/supabase/createClient';
 import COLORS from '@/styles/colors';
-import { SpacerDiv, H4Centered, QuestionsDiv } from '@/app/(auth)/styles';
+import { SpacerDiv, HorizontalDiv, H4Centered } from '@/app/(auth)/styles';
 import BigButton from '@/components/BigButton';
+import Button from '@/components/Button';
 
 export default function SignUp() {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  // const [errorMessage, setErrorMessage] = useState('');
-  const { push } = useRouter();
+  const [emailSentCount, setEmailSentCount] = useState(0);
+  // const { push } = useRouter();
   const handleSignUp = async () => {
     const { error } = await supabase.auth.signUp({
       email,
       password,
+      options: {
+        emailRedirectTo: 'http://localhost:3000/email-verified',
```

Comment:
Shall we make a global variable (similar to `COLORS`) that provide the address of the site? This will make the shift from development to production a lot easier (lawyers will likely not have a localhost running on their computer).

Reply:
save for next sprint?

Reply:
yes agree that we need this, and yes let's do it in a future sprint hehe